### DISCUSSION TITLE
iPhone Wallet shows Tokens, Mac Defi Wallet does not

### DISCUSSION THREAD

**Discussion by AlexandraKreisHelmstetter** [DeFiCh/app#1047] [Q&A] [answered]
I had the Defi Wallet first on my iPhone. And was curious how to see my LM details. 
I was recommended to pull up the app on my laptop. Which I did.
I synchronised the wallet first and then used my 24 seeds to open up the wallet. Now I don't have any value showing in my desktop App.
Can anyone point to what I did wrong - where?
Thanks

**Comment by izzycsy** [comment, marked as answer]
Hi @AlexandraKreisHelmstetter , at the moment, DeFiChain Wallet on mobile and DeFi App on Desktop are operate separately, the 24 words are unique to the respective wallets. If you would like to have your assets on either one of the app, you need send to your respective address.

**Comment by AlexandraKreisHelmstetter** [comment]
thanks Bella for your reply.

I have a problem opening the Desktop app. It says it can not load
blockchain.
My previous action to solve this problem were:

   1. reload from snapshot
   2. deinstall and re- install
   3. using the 24 passwords


with no results.

Can you help with this?
It would be appreciated as you can imagine I have about 500 Euros in there
that I would like to trade with.

Merci
Alexandra


On Thu, Oct 21, 2021 at 11:55 AM Bella ***@***.***> wrote:

> Hi @AlexandraKreisHelmstetter
> <https://github.com/AlexandraKreisHelmstetter> , at the moment, DeFiChain
> Wallet on mobile and DeFi App on Desktop are operate separately, the 24
> words are unique to the respective wallets. If you would like to have your
> assets on either one of the app, you need send to your respective address.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/DeFiCh/app/discussions/1047#discussioncomment-1513431>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AV2YXWZOVIWFXJJBXTFIZHTUH7PR7ANCNFSM5E7CH6YA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
Alexandra Kreis
Yoga - Coaching - Massage
Book a conversation
<https://app.acuityscheduling.com/schedule.php?owner=18298669?>

Mobil:+49 (0)162 2596875
www.alexandrakreis.com
Don't miss out on all the superb Invitations and freebies and sign up here
<http://madmimi.com/signups/376028/join>
